.github/workflows/canary.yml

```diff
@@ -60,10 +62,10 @@ jobs:
             ${{ runner.os }}-node-${{ steps.npm-timed-expiration.outputs.prefix }}-
 
       - name: install main
-        run: 'npm ci --ignore-scripts --force'
+        run: npm ci --ignore-scripts --force --no-fund --no-audit
         working-directory: ./main
       - name: install project
-        run: '[ -f package-lock.json ] && npm ci --ignore-scripts --force || npm install --ignore-scripts --force'
+        run: '[ -f package-lock.json ] && npm ci --ignore-scripts --force --no-fund --no-audit || npm install --ignore-scripts --force --no-fund --no-audit'
```

Comment:
I ensured we install consistently across all actions: `--ignore-scripts --force --no-fund --no-audit`

* `--ignore-scripts` for security and I guess speed
* `--force` in case we get one of those peer dependency issues again
* `--no-fund` because funding reminders are pointless in CI
* `--no-audit` because audits are handled elsewhere in CI and only adds noise (and maybe gets a tiny fraction quicker)